Workspaces Sharing Docs (#697)
elevenlabs/elevenlabs-docs: fern/docs/pages/product-guides/administration/workspaces/sso.mdx

  </Step>
</Steps>

## FAQ

<AccordionGroup>
  <Accordion title="Microsoft Entra Identifier / Azure AD - SAML">
    What shall I fill for Identifier (Entity ID)?
    - Use Service Provider Entity Id

    What shall I fill for Reply URL (Assertion Consumer Service) URL in SAML?
    - Use Redirect URL

    What is ACS URL?
    - Same as Assertion Consumer Service URL

    Which fields should I use to provide ElevenLabs?
    - Use _Microsoft Entra Identifier_ for IdP Entity ID
    - Use _Login URL_ for IdP Sign-In URL

  </Accordion>
  <Accordion title="I am getting the error 'Unable to login with saml.workspace...'">

>     - One known error: make sure you have `<saml:NameID>` having email address as the value inside `<saml:Subject>` in the SAML response. We use Google Cloud as our identity platform and it expects the `<saml:Subject>` and `<saml:NameID>` elements in responses from the identity provider. If you don't define values for these elements when configuring your provider, the SAML assertion fails.

Reviewer: This whole paragraph needs rewording since it's overly confusing. In general we shouldn't use first person pronouns in the docs.

```suggestion
    - One known error: make sure you have `<saml:NameID>` having email address as the value inside `<saml:Subject>` in the SAML response. ElevenLabs uses Google Cloud as the identity platform, it expects the `<saml:Subject>` and `<saml:NameID>` elements in responses from the identity provider. If you don't define values for these elements when configuring your provider, the SAML assertion will fail.
```
